packages/apollo/package.json

```diff
@@ -60,6 +60,7 @@
     "heroku-cli-util": "^8.0.9",
     "listr": "^0.14.1",
     "lodash": "^4.17.10",
+    "moment": "^2.24.0",
```

Comment:
Can we use something lighter than moment like date-fns?

Reply:
Looked for anything to support durations with the ease-of-use + granularity that ISO 8601 gives us and no love, + discussed offline. Going to leave as is. If having moment gets to heavy, we can write a duration parser.